src/main/java/frc/robot/commands/DriveToReefCommand.java

```diff
@@ -95,9 +100,15 @@ public void initialize() {
     public void execute() {      
       m_drivetrain.setControl(driveRequest
         .withSpeeds(new ChassisSpeeds(
-          xController.calculate(m_drivetrain.getState().Pose.getX(), m_targetTranslation.getX()),
-          yController.calculate(m_drivetrain.getState().Pose.getY(), m_targetTranslation.getY()),
+          xController.calculate(m_drivetrain.getState().Pose.getX(), m_targetTranslation.getX()) + xController.getSetpoint().velocity,
```

Comment:
As a style point in the future, I think it's less confusing if you assign sums like these to a local variable (and allows you to easily write log statements in the future).